sklearn/utils/tests/test_utils.py

```diff
@@ -126,6 +130,28 @@ def test_pinvh_simple_complex():
     assert_almost_equal(np.dot(a, a_pinv), np.eye(3))
 
 
+def test_arpack_eigsh_initialization():
+    '''
```

Comment:
We prefer to start tests with comments, not docstrings, so we can see the name of the test that is failing.